src/test/resources/org/jetbrains/research/ml/ast/transformations/deadcode/data/in_1.py

```diff
@@ -0,0 +1,8 @@
+def main(x):
```

Comment:
It would be easier to check your test coverage if the test files were split into logical cases, for example, 'if' statements, 'while' loops, and so on, and divided into the corresponding directories. Make sure they match your list of the supported cases you should mention in the doc file.

Reply:
Yes, I agree. The `getInAndOutArray` function get all nested files from the root (`data` here) directory. As a result, you should not change the test's logic, you should only split the files into several folders.